modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -646,6 +646,7 @@ export interface SendManyOptions extends PrebuildAndSignTransactionOptions {
     feeLimit?: string;
     data?: string;
     tokenName?: string;
+    tokenData?: TokenTransferRecipientParams;
```

Comment:
what is the difference between data and tokenData ?
isn't it confusing for the user ?

Reply:
`data` field is being populated internally [here](https://github.com/BitGo/BitGoJS/blob/master/modules/sdk-core/src/bitgo/wallet/wallet.ts#L2425).
`tokenData` field of type `TokenTransferRecipientParams` is less confusing for the users than the `data` field of type string. The `data` field is being used specifically for `eth`-like coins.